Apps/WebAPI/sdLitica.WebAPI/Models/TimeSeries/TimeSeriesMetadataModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using sdLitica.Entities.TimeSeries;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace sdLitica.WebAPI.Models.TimeSeries
+{
+    public class TimeSeriesMetadataModel
```

Comment:
Please, add documentation for class and its public members

Reply:
added